### PR TITLE
Pali wallet v2 support

### PR DESCRIPTION
**Pull request by osiastedian**

### CODE REVIEW

File: blockchain/NevmRolluxBridge/config/chainsUseDapp.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    blockExplorerUrl: "https://tanenbaum.io",
+}
+
+export const RolluxChain: Chain = {
+    chainId: 57000,
+    chainName: "Rollux Bedrock Testnet",
+    isLocalChain: false,
+    isTestChain: false,
+    nativeCurrency: {
+        name: "bSyscoin",
```
Comment (sidhujag) [inline blockchain/NevmRolluxBridge/config/chainsUseDapp.ts:52]: just Syscoin

File: blockchain/NevmRolluxBridge/config/chainsUseDapp.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        symbol: "bSYS",
```
Comment (sidhujag) [inline blockchain/NevmRolluxBridge/config/chainsUseDapp.ts:53]: SYS

File: blockchain/NevmRolluxBridge/config/networks.ts [hunk 3]
```diff
@@ -21,20 +23,23 @@ export const networks: { [key: string]: NetworkData } = {
...
         rpcAddress: 'https://rpc.tanenbaum.io',
         name: 'L1Dev',
         explorerUrl: 'https://tanenbaum.io',
         chainId: 5700,
+        layer: 1,
         contracts: contractsDev.l1_dev,
     },
     L2Dev: {
-        rpcAddress: 'https://bedrock.rollux.com:9545/',
+        rpcAddress: 'https://rpc-bedrock.rollux.com/',
```
Comment (sidhujag) [inline blockchain/NevmRolluxBridge/config/networks.ts:38]: this isn't right, right?

**Review by sidhujag — CHANGES_REQUESTED** [review]

File: blockchain/NevmRolluxBridge/config/networks.ts [hunk 4]
```diff
@@ -21,20 +23,23 @@ export const networks: { [key: string]: NetworkData } = {
...
         name: 'RolluxBedrockDev',
-        explorerUrl: 'https://blockscout.com/optimism/bedrock-alpha/',
+        explorerUrl: 'https://explorer.testnet.rollux.com',
```
[the remark this answers is not in the document]
Comment (eskrano) [reply blockchain/NevmRolluxBridge/config/networks.ts:40]: @sidhujag right version will be at bridge-main branch for all configs and other things. I dont think usedapp is used for UTXO bridge. Better to just avoid config files old code for now.

File: pages/index.tsx [hunk 5]
```diff
@@ -80,7 +80,7 @@ const Home: NextPage = () => {
               <Grid item md={6}>
                 <Box>
                   <NextImage
-                    className="animation"
+                    className="animatiogn"
```
Comment (sidhujag) [inline pages/index.tsx:83]: typo?

**Pull request merged**